### PR TITLE
Job page redesign and refactoring

### LINKED ISSUE #368: Redesign of job details page
**Comment:**
Is has become clear that the interface is not that intuitive to the users (noted by the user behavior.)

New design suggestion:
_Not applied yet:_
job details view expanded
job details view

_Have applied:_
job details view - have applied expanded
job details view - have applied

### LINKED ISSUE #406: Use GET /jobs/:job_id/users/:user_id/job-user
https://github.com/justarrived/just_match_api/pull/1047

### LINKED ISSUE #408: Refactor the job details view
Should be refactored into a page with components using semantic.

### PR DESCRIPTION
**Pull request by bex1**
closes #408 
closes #368
closes #406 

Job page redesign.
Refactored properly into reusable selft contained subcomponents as usual.

Job page went from around 150 lines of html to 20-ish.

Ready for deploy.

### CODE REVIEW

**Comment by bex1** [comment]
NOTE: Circleci is still failing due to old yarn version on circleci...

File: src/app/components/sections/job-comments-section/job-comments-section.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {Component} from '@angular/core';
+import {Input} from '@angular/core';
+import {Job} from '../../../models/api-models/job/job'
+
+@Component({
+  selector: 'job-comments-section',
+  template: `
+    <div class="ui basic center aligned segment">
+      <i class="ui big circular inverted pink wechat icon"></i>
```
Comment (buren) [inline src/app/components/sections/job-comments-section/job-comments-section.component.ts:9]: Is that the actual wechat logo? That could be quite misleading for a user that is familiar with the brand.. Plus we should use a generic icon and not a company logo ;)

**Review by buren — COMMENTED** [review]
Wow! Great work 🙇

File: src/app/components/sections/job-comments-section/job-comments-section.component.ts (hunk 1 above)
Comment (bex1) [reply src/app/components/sections/job-comments-section/job-comments-section.component.ts:9]: Ah yes, it probably is. I will change it.
Thanks!

Comment (buren) [reply src/app/components/sections/job-comments-section/job-comments-section.component.ts:9]: 👍

**Pull request merged**